fix(content): Fix outdated news entry about human drones (#10077)
endless-sky/endless-sky: data/human/news.txt

			"I don't get how people could get worked up over what ship to use. As long as I'm in my trusty Protector, I'm happy."
			"Ugh. Why are fighter pilots so gung-ho about charging right into the guns of their enemies? At the very least, don't make me waste hundreds of thousands of credits replacing my ships when you kill yourself."
			"Don't you think it's strange that the Navy lets you buy their offensive fighters but not any of their other ships?"

> 			"I wonder why shipyards haven't capitalized on making alternatives to the Navy combat drone. Probably because they are too expensive for regular use, I guess."

Reviewer: How about this then?
```suggestion
			"I wonder why shipyards haven't capitalized on making alternatives to the Navy combat drone. Probably because they don't last as long as fighters, I guess."
```
